zerver/openapi/zulip.yaml

```diff
@@ -6161,7 +6163,26 @@ paths:
                         type: integer
                         description: |
                           The unique ID assigned to the sent message.
-                    example: {"msg": "", "id": 42, "result": "success"}
+                      automatic_new_visibility_policy:
+                        type: integer
+                        enum:
+                          - 2
+                          - 3
+                        description: |
+                          The new visibility policy value due to the `automatically_follow_topics_policy`
+                          and `automatically_unmute_topics_in_muted_streams_policy` user settings during the
+                          send message action.
+
+                          Only present if there is a change in the visibility policy.
```

Comment:
A few suggestions for this:
* This documentation doesn't specify which user is affected by the change.
* There's no link to /help/ documentation about visibility policies or these settings (might be a follow-up if they don't exist yet)
* Context on how a client might use this information is quite minimal.

I suggest: 
"If the message's sender had configured their [visibility policy settings](/help/user-topics) to potentially automatically follow or unmuted topics when sending messages, and one of these policies did in fact change the user's visibility policy for the topic where this message was sent, the new value for that user's visibility policy for the recipient topic.

Only present if the sender's visibility was in fact changed due.

Intended to be used by clients where the information is important to correctly decide whether to display a warning/noticing suggesting unmuting the topic after sending a message to a muted stream. Such a notice might feel spurious/confusing in the event that the act of sending the message had already resulted in the user unmuting or following the topic in question."

When writing API docs, it's always helpful to imagine someone writing pseudocode for client code based on the documentation without using any other knowledge about how that parameter works... and trying to pay attention to what facts one needs to know to do that.

Reply:
Extended the last paragraph a bit:

> Intended to be used by clients where it is important to explicitly display a notice that the topic's visibility policy has changed automatically due to the message sent. The information is also important to correctly decide whether to display a warning or notice suggesting to unmute the topic after sending a message to a muted stream. Such a notice might feel confusing in the event that the act of sending the message had already resulted in the user unmuting or following the topic in question.

Reply:
Looks great!